autoload/SpaceVim/lsp.vim

```diff
@@ -8,7 +8,38 @@
 
 scriptencoding utf-8
 
-if has('nvim')
+if g:spacevim_autocomplete_method ==# 'coc'
+  " use coc.nvim
+  let s:coc_language_servers = {}
+  function! SpaceVim#lsp#reg_server(ft, cmds) abort
```

Comment:
1. the argument `cmds` maybe just a string. 
2. if the cmds is a list, the first argv maybe the whole path of the executable

Reply:
As to the first point, I think the the documentation doesn't mention the possibility of cmds being a string.
Furthermore, I found this function which also assumes `cmds` to be an array

https://github.com/SpaceVim/SpaceVim/blob/9634851a1dc6be2b2aaa124c105ccc010a9b70fc/autoload/SpaceVim/layers/lsp.vim#L103-L120

Reply:
As to the second point, please see comments below